aws/security_plugins/db2-aws-iam/install_aws_sdk.sh

```diff
@@ -0,0 +1,15 @@
+
```

Comment:
It is very concerning that we are suggesting to users (and guiding them through the process) of rebuilding and replacing the core Amazon SDK libraries that we ship with Db2 with their own.  While samples are 100% unsupported, this process makes a user's Db2 installation entirely unsupported.

It would be much more palatable if the rebuilt Amazon SDK was copied into the security plugin directory, and hard-coded into the build of the security plugins themselves. This way the integrity of the Amazon SDK libraries shipped with Db2 is preserved.